nextflow.config

```diff
@@ -65,14 +84,15 @@ profiles {
         shifter.enabled         = false
         charliecloud.enabled    = false
         apptainer.enabled       = false
-        docker.runOptions       = '-u $(id -u):$(id -g)'
+        docker.runOptions       = params.use_gpus ? '--gpus all' : '-u $(id -u):$(id -g)'
```

Comment:
I know. Actually, I don't think this was used to get the pipeline running 🤔 Because the Platform won't use the `docker` profile unless specified. I'll check and remove it.

Reply:
Just realised I don't actually know where to find the `docker run` command executed by NF in the `.command.*` files. It's not in `.command.run`. Maybe something that is controlled by Fusion. 

Going to leave it as it is for now and we can remove if we get any issues reported.